unittests/CppInterOp/ScopeReflectionTest.cpp

```diff
@@ -772,7 +772,11 @@ template<class T> constexpr T pi = T(3.1415926535897932385L);
   VarTemplateDecl* VDTD1 = VD->getSpecializedTemplate();
   EXPECT_TRUE(VDTD1->isThisDeclarationADefinition());
 #if CLANG_VERSION_MAJOR > 13
+#if CLANG_VERSION_MAJOR > 18
+  TemplateArgument TA1 = (*VD->getTemplateArgsAsWritten())[0].getArgument();
+#else
   TemplateArgument TA1 = (*VD->getTemplateArgsInfo())[0].getArgument();
+#endif // CLANG_VERSION_MAJOR
```

Comment:
```suggestion
#if CLANG_VERSION_MAJOR <= 18
  TemplateArgument TA1 = (*VD->getTemplateArgsInfo())[0].getArgument();
#else
  TemplateArgument TA1 = (*VD->getTemplateArgsAsWritten())[0].getArgument();
#endif // CLANG_VERSION_MAJOR
```

This way when we decide to drop some day the support up to llvm 18 we will find this with git grep and clean it ;)